app/styles/crate/version.module.scss

```diff
@@ -5,6 +5,26 @@
     top: 4px;
 }
 
+ul.quick-links {
```

Comment:
do you need the `ul` specifier?

Reply:
not strictly necessary. do you want me to remove it?

Reply:
removed